submit_cgap/submission.py

```diff
@@ -924,7 +946,7 @@ def do_uploads(upload_spec_list, auth, folder=None, no_query=False, subfolders=F
                     uploader_wrapper,
                     folder,
                     auth,
-                    recursive=subfolders,
+                    recursive=subfolders
```

Comment:
When using vertical format, pretty much always use trailing commas in python code that is vertically listed. That way in the future to add a line, you only have to add one new line, not change the previous line to add that comma.
```suggestion
                    recursive=subfolders,
```
Even in the case a bit farther above (and there's another a little farther below) where you had this:
```
        file_metadata = wrapped_upload_file_to_uuid(
            filename=file_path, uuid=uuid, auth=auth
        )
```
you should have a trailing comma after `auth`. Nothing to change in this case because you're moving it all onto a single case, and in that horizontal arrangement the trailing comma is not conventionally used. But if you'd not been consolidating it, I'd have suggested adding a trailing comma, as I always do in those long
multi-line imports. It gives more flexibility of how to add stuff later with minimal diffs.